modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/UpdateObjectEntryObjectActionExecutorImpl.java

```diff
@@ -41,16 +62,26 @@ public void execute(
 			JSONObject payloadJSONObject, long userId)
 		throws Exception {
 
-		_objectEntryLocalService.updateObjectEntry(
-			userId,
-			GetterUtil.getLong(parametersUnicodeProperties.get("classPK")),
-			HashMapBuilder.<String, Serializable>put(
-				parametersUnicodeProperties.get("objectFieldName"),
-				parametersUnicodeProperties.get("objectFieldValue")
-			).build(),
-			new ServiceContext() {
+		ObjectDefinition sourceObjectDefinition =
+			_objectDefinitionLocalService.fetchObjectDefinition(
+				payloadJSONObject.getLong("objectDefinitionId"));
```

Comment:
@marcelabc I think this can be just `objectDefinition`, since we don't have the `targetObjectDefinition` here